Cargo.toml

```diff
@@ -15,7 +15,7 @@ edition = "2018"
 default = ["native-client", "middleware-logger"]
 native-client = ["curl-client", "wasm-client"]
 hyper-client = ["hyper", "runtime", "runtime-raw", "runtime-tokio" ]
-curl-client = ["chttp"]
+curl-client = ["isahc"]
```

Comment:
This is a breaking change we can't merge until we're comfortable releasing a 2.0. I feel like we may need to revert this and log it so we don't forget to update it at a later time (:

Reply:
Ah nevermind, I totally misread. We're good here!